src/main/java/org/motechproject/mots/mapper/UuidMapper.java

```diff
@@ -1,7 +1,9 @@
 package org.motechproject.mots.mapper;
 
 import java.util.UUID;
+import org.mapstruct.Mapper;
 
+@Mapper(componentModel = "spring")
```

Comment:
are you sure this will not break any other mappers using this one?

Reply:
As above